lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -361,11 +360,6 @@
                     })
                 });
             });
-            this.$btn_download_blank_lti.click(function() {
-                var url;
-                url = _this.$btn_download_blank_lti.data('endpoint');
```

Comment:
This was a copy/paste fail from my origin changes where I mistakenly edited a compiled JS file, instead of the main source file.

`_this` isn't defined in this context :\

Reply:
DOH!